### ISSUE TITLE
 Error: Invalid header: Does not start with Cr24

### ISSUE THREAD

**Issue by moetanahy** [frappe/books#190]
When following the installation instructions, when running the command
```
yarn electron:serve
```
I get the following error when running
```
INFO  Launching Electron...
Uncaught (in promise) Error: Invalid header: Does not start with Cr24
    at crxToZip (webpack:///./node_modules/unzip-crx/dist/index.js?:32:15)
    at eval (webpack:///./node_modules/unzip-crx/dist/index.js?:57:32)
    at callHanler (webpack:///./node_modules/yaku/lib/yaku.js?:719:16)
    at tryCatcher (webpack:///./node_modules/yaku/lib/yaku.js?:436:32)
    at eval (webpack:///./node_modules/yaku/lib/yaku.js?:575:38)
    at flush (webpack:///./node_modules/yaku/lib/yaku.js?:478:17)
    at processTicksAndRejections (internal/process/task_queues.js:82:9)
```
When doing some research, I came across this - https://github.com/electron-react-boilerplate/electron-react-boilerplate/issues/2577.  This states that the error might be related to the version of electron-devtools-installer and that bumping the version to ^3.1.1 might solve this issue.

However, when I upgrade the version and rerun, it causes more issues and the error remains.

**Comment by azrikahar** [comment]
Faced the same issue today. After trying to upgrade several packages, I ended up just dropping every changes I did, and do just this single thing to at least make it start up: 

Comment out this line here: https://github.com/frappe/books/blob/master/src/background.js#L122

Definitely just a stop-gap measure and we lose vue devtools as a result.

**Comment by ankitsinghaniyaz** [comment]
This can be fixed after upgrading some of the packages.

**Comment by 18alantom** [comment]
@moetanahy build-related fixes were merged through #216. You can retry setting it up, it should work now.
